content/docs/reference/genesis/_index.md

```diff
@@ -122,84 +129,130 @@ The `alloc` object is used to issue native coin and allows pre-deployment of sma
-          "bondedStake": 10000000000000000000000,
           "enode": "enode://e3b8ea9ddef567225530bcbae68af5d46f59a2b39acc04113165eba2744f6759493027237681f10911d4c12eda729c367f8e64dfd4789c508b7619080bb0861b@35.189.64.207:30303",
-          "treasury": "0xf1859D9feD50514F9D805BeC7a30623d061f40B7"
+          "treasury": "0xf1859D9feD50514F9D805BeC7a30623d061f40B7",
+          "oracleAddress": "0xd54ba484243c99CE10f11Bc5fb24cCc728ba060D",
+          "bondedStake": 10000000000000000000000
         },
         {
-          "bondedStake": 10000000000000000000000,
           "enode": "enode://00c6c1704c103e74a26ad072aa680d82f6c677106db413f0afa41a84b5c3ab3b0827ea1a54511f637350e4e31d8a87fdbab5d918e492d21bea0a399399a9a7b5@34.105.163.137:30303",
-			"treasury": "0x1B441084736B80f273e498E646b0bEA86B4eC6AB"
+          "treasury": "0x1B441084736B80f273e498E646b0bEA86B4eC6AB",
+          "oracleAddress": "0xF99bC17d7db947Bf4E7171519D678882FF3Dcb8d",
+          "bondedStake": 10000000000000000000000
         },
         {
-          "bondedStake": 10000000000000000000000,
           "enode": "enode://dffaa985bf36c8e961b9aa7bcdd644f1ad80e07d7977ce8238ac126d4425509d98da8c7f32a3e47e19822bd412ffa705c4488ce49d8b1769b8c81ee7bf102249@35.177.8.113:30308",
-          "treasury": "0xB5C49d50470743D8dE43bB6822AC4505E64648Da"
+          "treasury": "0xB5C49d50470743D8dE43bB6822AC4505E64648Da",
+          "oracleAddress": "0x89f2CabCA5e09f92E49fACC10BBDfa5114D13113",
+          "bondedStake": 10000000000000000000000
         },
         {
-          "bondedStake": 10000000000000000000000,
           "enode": "enode://1bd367bfb421eb4d21f9ace33f9c3c26cd1f6b257cc4a1af640c9af56f338d865c8e5480c7ee74d5881647ef6f71d880104690936b72fdc905886e9594e976d1@35.179.46.181:30309",
-          "treasury": "0x31e1dE659A26F7638FAaFEfD94D47258FE361823"
+          "treasury": "0x31e1dE659A26F7638FAaFEfD94D47258FE361823",
+          "oracleAddress": "0x7CF62D2C8314445Df0bF3F322f84d3BF785e4aeF",
+          "bondedStake": 10000000000000000000000
         },
         {
-          "bondedStake": 10000000000000000000000,
           "enode": "enode://a7465d99513715ece132504e47867f88bb5e289b8bca0fca118076b5c733d901305db68d1104ab838cf6be270b7bf71e576a44644d02f8576a4d43de8aeba1ab@3.9.98.39:30310",
-          "treasury": "0xe22617BD2a4e1Fe3938F84060D8a6be7A18a2ef9"
+          "treasury": "0xe22617BD2a4e1Fe3938F84060D8a6be7A18a2ef9",
+          "oracleAddress": "0xD689E4D1061a55Fd9292515AaE9bF8a3C876047d",
+          "bondedStake": 10000000000000000000000
         }
       ]
+    },
+    "oracle": {
+      "symbols":[
+            "NTN/USD",
+            "NTN/AUD",
+            "NTN/CAD",
+            "NTN/EUR",
+            "NTN/GBP",
+            "NTN/JPY",
+            "NTN/SEK",
+            "AUD/USD",
+            "CAD/USD",
+            "EUR/USD",
+            "GBP/USD",
+            "JPY/USD",
+            "SEK/USD",
+            "ATN/USD",
+            "NTN/USD",
+            "NTN/ATN"
+         ],
```

Comment:
AFAIK this symbols list may be more representative:

```suggestion
      "symbols":[
            "AUD/USD",
            "CAD/USD",
            "EUR/USD",
            "GBP/USD",
            "JPY/USD",
            "SEK/USD",
            "ATN/USD",
            "NTN/USD",
         ],
```

Reply:
Yes; good catch